src/server/pegasus_server_impl.h

```diff
@@ -206,6 +206,7 @@ class pegasus_server_impl : public pegasus_read_service
     // return 1 if value is appended
     // return 2 if value is expired
     // return 3 if value is filtered
+    // return 4 if key is not pass partition_hash validation
```

Comment:
Better to refactor these code and define them as an enum class

Reply:
Sure~ I will refactor in next pull request, this pr only contains functional code.